.gitlab/pipeline/test.yml

```diff
@@ -312,9 +312,8 @@ test-frame-support:
     # Ensure we run the UI tests.
     RUN_UI_TESTS: 1
   script:
-    - time cargo test --locked -p frame-support-test --features=frame-feature-testing,no-metadata-docs,try-runtime,experimental --manifest-path ./substrate/frame/support/test/Cargo.toml
-    - time cargo test --locked -p frame-support-test --features=frame-feature-testing,frame-feature-testing-2,no-metadata-docs,try-runtime,experimental --manifest-path ./substrate/frame/support/test/Cargo.toml
-    - SUBSTRATE_TEST_TIMEOUT=1 time cargo test -p substrate-test-utils --release --locked -- --ignored timeout
```

Comment:
Apparently it was running this test: https://github.com/paritytech/polkadot-sdk/blob/22393d51e4a90799fd2541b696e413eab0a9e692/substrate/test-utils/tests/basic.rs#L41  
Either was, it does not belong into the FRAME CI job.